apps/calendar/locales/calendar.en-US.properties

```diff
@@ -44,7 +44,7 @@ advanced-settings-add-account=Add Account
 field-user.placeholder=Username
 field-password.placeholder=Password
 field-full-url.placeholder=URL
-modify-account-setting-up=Please wait while your account is set up
+modify-account-setting-up=Setting up account
```

Comment:
Unfortunately for our translators to be aware of this change the key needs to change as well. You could either change it to something like ```modify-account-setting-up-2``` or ```setting-up-account```, I think either could work. (You'll also need to update the key anywhere we use it in calendar code).